plugins/modules/system/seport.py

```diff
@@ -113,7 +113,9 @@
 
 
 def get_runtime_status(ignore_selinux_state=False):
-    return True if ignore_selinux_state is True else selinux.is_selinux_enabled()
+    if ignore_selinux_state is False:
+        return selinux.is_selinux_enabled()
+    return True
```

Comment:
I would do it the other way around:
```suggestion
    if ignore_selinux_state is True:
        return True
    return selinux.is_selinux_enabled()
```
Quick exit if `ignore_selinux_state` is `True`, and the 'proper' code as the main part of the function. (Not that the proper code here is much longer than the quick exit...)

Reply:
Kinda personal preference, I suppose. I am bothered by code like:
```python
if x is True:
    return True
```
In terms of compilation, both will likely produce the exact same low level code.

Reply:
You can also change it to `if x: return True` :-) Or change the whole function body to `return ignore_selinux_state or selinux.is_selinux_enabled()` ;)

Reply:
I like that.